src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const kindOfAction of actions) {
+    switch (kindOfAction.type) {
+      case 'addProperties':
+        Object.assign(state, kindOfAction.extraData);
+        break;
+      case 'removeProperties':
+        kindOfAction.keysToRemove.forEach(el => {
+          delete state[el];
+        });
+        break;
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+    }
```

Comment:
let's think about the readability of our code and add spaces between cases

Reply:
👌 fixed